src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -1266,9 +1266,40 @@ public QueryOperationResponse<T> LoadProperty<T>(object entity, string propertyN
         public Uri GetReadStreamUri(object entity)
         {
             Util.CheckArgumentNull(entity, "entity");
-            EntityDescriptor box = this.entityTracker.GetEntityDescriptor(entity);
+            EntityDescriptor box = this.GetEntityDescriptorForStreamEntity(entity);
             return box.ReadStreamUri;
         }
+        /// <summary>

```

Comment:
```suggestion

        /// <summary>
```
This actually produces a warning
```SA1513 : CSharp.Layout : Statements or elements wrapped in curly brackets must be followed by a blank line.```